tests/Charge/ManageChargesTest.php

```diff
@@ -0,0 +1,75 @@
+<?php
+
+namespace Laravel\Cashier\Tests\Charge;
+
+use Laravel\Cashier\Charge\FirstPaymentChargeBuilder;
+use Laravel\Cashier\Charge\MandatedChargeBuilder;
+use Laravel\Cashier\FirstPayment\Actions\AddBalance;
+use Laravel\Cashier\FirstPayment\Actions\AddGenericOrderItem;
+use Laravel\Cashier\FirstPayment\FirstPaymentBuilder;
+use Laravel\Cashier\Mollie\Contracts\CreateMollieCustomer;
+use Laravel\Cashier\Mollie\Contracts\CreateMolliePayment;
+use Laravel\Cashier\Mollie\Contracts\GetMollieCustomer;
+use Laravel\Cashier\Mollie\Contracts\GetMollieMandate;
+use Laravel\Cashier\Payment;
+use Laravel\Cashier\Tests\BaseTestCase;
+use Laravel\Cashier\Tests\Fixtures\User;
+use Mollie\Api\MollieApiClient;
+use Mollie\Api\Resources\Customer;
+use Mollie\Api\Resources\Mandate;
+use Mollie\Api\Resources\Payment as MolliePayment;
+use Mollie\Api\Types\SequenceType;
+
+class ManageChargesTest extends BaseTestCase
+{
+    protected function setUp(): void
+    {
+        parent::setUp();
+        $this->withPackageMigrations();
+
+    }
+
+    /** @test */
+    public function useFirstPaymentChargeBuilder()
```

Comment:
Naming can be a bit more verbose...

How about `usingFirstPaymentChargeBuilderWhenNoMandate`?